### ISSUE TITLE
[Request] turtle.use() function

### ISSUE THREAD

**Issue by vpontin** [SquidDev-CC/CC-Tweaked#58]
Little summary:

Since 1.9.x [grass paths](https://minecraft.gamepedia.com/Grass_Path) were added to Minecraft. They are created using (right-click) a shovel on a grass block (or other dirt blocks if using [a mod](https://minecraft.curseforge.com/projects/dirt2path)).

The actual behavior of turtles to create it, according to SquidDev in discord is to use `[turtle.dig()](http://computercraft.info/wiki/Turtle.dig)` but since shovel is also the preferred way to actually dig dirt-like blocks it cause somekind a ambiguous and confuse case for newcomers at turtle scripting. Similar thing is with `turtle.dig()` for hoes.

So i thought: if a proper `use()` function could be implemented for turtles scripting would be more organized without ambiguous cases like these.

Sorry if i wasnt much clear, but this is a idea sprouting my mind.

**Comment by Wendelstein7** [comment]
Good idea, this would render using turtle.place() to fill a bucket obsolete, so that we could use turtle.use() with a bucket instead to fill them. Similarly, turtle.use() could perhaps also throw a throw-able potion, press a button block and more, however this is subject to thought, as this may render turtles too powerful where we might want other mods (such as Plethora) to provide techniques for achieving these things like pressing buttons and throwing potions.

along with `turtle.use()`, obviously `turtle.useUp()` and `turtle.useDown()` would also be implemented.

**Comment by vpontin** [comment]
Moved to https://github.com/dan200/ComputerCraft/issues/564
